google-cloud-spanner/clirr-ignored-differences.xml

```diff
@@ -11,4 +11,19 @@
     <className>com/google/cloud/spanner/spi/v1/SpannerRpc</className>
     <method>* asyncDeleteSession(*)</method>
   </difference>
+  <difference>
```

Comment:
FYI @chingor13 I got [this](https://source.cloud.google.com/results/invocations/08e7c448-6909-4ebb-a3ad-5d561fae985a/targets/cloud-devrel%2Fclient-libraries%2Fjava%2Fjava-spanner%2Fpresubmit%2Fclirr/log) binary compatibility error as a result of the auto-generated code so I added them to the list of ignored differences here. Please let me know if you have any concerns about this.